package.json

```diff
@@ -10,10 +10,10 @@
     "prebuild": "rimraf dist",
     "validate": "npm-run-all --parallel lint cover build test:configs --sequential check-coverage",
     "build": "babel -d dist src",
-    "cover": "nyc --reporter=lcov --reporter=text --reporter=html npm run test",
+    "cover": "nyc npm test --reporter=lcov --reporter=text --reporter=html",
```

Comment:
Is this really necessary? It is counter-intuitive to put the command to cover before the options for the coverage...